### DISCUSSION TITLE
Invalid PGP singatures

### DISCUSSION THREAD

**Discussion by magical-heyrovsky** [simple-login/app#1448] [Q&A]
I had PGP enabled since that feature released (so at least 1+ years) and in all that time I always received some emails where the PGP signature was wrong. I think it depends on the sender. For some senders it's always valid, for others it's almost always invalid. For example, 9 out of 10 emails I get from `*@patreon.com` have an invalid signature. Another example: Today I created an account on [etail.market](https://etail.market/) and bought a game, and again, every single email  I received during that process (3-4 emails) had an invalid signature. Is that normal? Overall about 5% of all emails that go through SimpleLogin have an invalid signature.

**Comment by 4b15ed24f576** [comment]
Hi there, could I ask you to contact our support team please?

You can do so by creating a ticket here: https://app.simplelogin.io/dashboard/support

This will help us to better assist you when using our PGP encryption feature.
